local-setup/scripts/hot-deploy.sh

```
@@ -0,0 +1,147 @@
+        exit 1
+    fi
+}
+
+cmd_backend() {
+    log "building backend/pgr-services jar..."
+    mvn -f "$REPO_ROOT/backend/pgr-services/pom.xml" clean package -DskipTests
+
+    require_container digit-pgr-services-1
+
+    local jar
+    jar="$(ls "$REPO_ROOT"/backend/pgr-services/target/pgr-services-*.jar 2>/dev/null | head -1)"
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)